pkg/trace/traceutil/span.go

```diff
@@ -40,6 +42,20 @@ func IsMeasured(s *pb.Span) bool {
 	return s.Metrics[measuredKey] == 1
 }
 
+// IsPartialSnapshot returns true if the span is a partial snapshot (has metric _dd.partial_version >= 0)
+func IsPartialSnapshot(s *pb.Span) bool {
+	v, ok := s.Metrics[partialVersionKey]
+	return ok && v >= 0
+}
+
+// SetPartialSnapshotVersion set the version of partial snapshot span.
```

Comment:
I don't think it's worth adding this helper in the API of this package only to be used in a test.

Reply:
Yes. It's just for testing. I'm changing accordingly